moveit2_rolling-ci.repos

```diff
@@ -0,0 +1,17 @@
+repositories:
+  geometric_shapes:
+    type: git
+    url: https://github.com/ros-planning/geometric_shapes
+    version: 2.1.2
+  moveit_msgs:
+    type: git
+    url: https://github.com/ros-planning/moveit_msgs
+    version: 2.2.0
+  moveit_resources:
+    type: git
+    url: https://github.com/ros-planning/moveit_resources
+    version: 2.0.3
+  launch_param_builder:
+    type: git
+    url: https://github.com/PickNikRobotics/launch_param_builder
+    version: main
```

Comment:
There is a tagged released version of this.  (it just hasn't been synced yet)

Secondly, don't you need to add ros2_control into this second ci one?

Reply:
ros2_control should not be required since the fix in moveit_resources is the reason we were failing in the first place. With moveit_resources set to the released version, the released version of ros2_control still works.